tests/dags/test_issue_1225.py

```diff
@@ -31,6 +33,10 @@
 def fail():
     raise ValueError('Expected failure.')
 
+def delayed_fail():
```

Comment:
Sleeping in tests is an antipattern, usually you want to want to use concurrency mechanisms instead. Can you put a TODO documenting this test so we can fix it later?